chrome/scripts.js

```diff
@@ -0,0 +1,19 @@
+var targetDate = new Date("Nov 4, 2021").getTime();
```

Comment:
One of the requirement of the issue is to have a function that can change the timer's targetDate. Also please avoid making global variables as many people will be working on this code, and polluting the global namespace will make the code harder to understand for them.